pyaim/aimccp.py

```diff
@@ -0,0 +1,63 @@
+import json
+import ssl
+import urllib.parse
+import urllib.request
+
+
+class CCPPasswordREST(object):
+
+    # Runs on Initialization
+    def __init__(self, base_uri, verify=True):
+
+        # Declare Init Variables
+        self._base_uri = base_uri.rstrip('/')   # Example: https://pvwa.cyberarkexample.com
+        
+        if verify is False:
+            self._context = ssl.create_default_context()
+            self._context.check_hostname = False
+            self._context.verify_mode = ssl.CERT_NONE
+        else:
+            self._context = ssl.create_default_context()
+
+
+    # Checks that the AIM Web Service is available
+    def check_service(self):
+        try:
+            conn = urllib.request.urlopen(self._base_uri + '/AIMWebService/v1.1/aim.asmx', context=self._context)
+            status_code = conn.getcode()
+            conn.close()
+        except Exception as e:
+            return 'AIMWebService Not Found. Exception: {}'.format(e)
+            exit()
+
+        return 'AIMWebService Found. Status Code: {}'.format(status_code)
+
+    # Retrieve Account Object Properties using AIM Web Service
+    def GetPassword(self, appid=None, safe=None, objectName=None):
```

Comment:
`safe` needs a better name that's descriptive

Reply:
`safe` is actually the name of the Safe in CyberArk where the account is stored.  I don't think I can get any more descriptive than that.  CyberArk Users will understand what it is referencing.